frame/staking/src/lib.rs

```diff
@@ -1521,8 +1583,13 @@ decl_module! {
 		/// - Independent of the arguments. Insignificant complexity.
 		/// - Contains one read.
 		/// - Writes are limited to the `origin` account key.
+		/// --------
+		/// Base Weight: 16.53 µs
```

Comment:
small note/nit: I prefer separating weight from execution time on our benchmarking hardware and would not say or write "weight is 15 micro seconds"

Reply:
what does this comment mean? Weight, in the context of our benchmarks, is exactly equal to the execution time on our benchmarking hardware.

Reply:
yeah, in the context of the benchmarks it is
never mind, it's hair splitting anyway